ui.apps/src/main/content/jcr_root/apps/etoolbox-link-inspector/clientlibs/link-inspector-ui/js/console-ui.filter.js

```diff
@@ -18,12 +18,12 @@
  */
 (function (window, document, $, ELC, Granite, Coral) {
     'use strict';
-    var DIALOG_TITLE_LABEL = Granite.I18n.get('Filter Options');
-    var SUCCESS_DIALOG_TITLE_LABEL = Granite.I18n.get('Filter Options Applied');
-    var CANCEL_LABEL = Granite.I18n.get('Cancel');
-    var SUBMIT_FILTER_LABEL = Granite.I18n.get('Apply');
+    const DIALOG_TITLE_LABEL = Granite.I18n.get('Filter Options');
```

Comment:
To discuss: is creating the whole dialog content "on the fly" via JS better than creating a "usual" Granite console in XML? I mean something of this kind: https://github.com/exadel-inc/etoolbox-backpack/blob/1013118d19cc7b8085db176483fe9c2986372732/ui.apps/src/main/content/jcr_root/apps/etoolbox-backpack/admin/new/.content.xml#L237. The dialog and all the inputs are declared much like a "usual" AEM component dialog and stored in an XML file / node. 

Then you can "bring on" this node and its content as a modal window onto the screen with a button like this https://github.com/exadel-inc/etoolbox-backpack/blob/1013118d19cc7b8085db176483fe9c2986372732/ui.apps/src/main/content/jcr_root/apps/etoolbox-backpack/admin/new/.content.xml#L79

Reply:
Each time with this method, will a new request be made for the component? or only once during the first use?